src/Agent.Plugins/TestResultParser/TraceLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Agent.Plugins.Log.TestResultParser.Contracts;
+using Agent.Sdk;
+
+namespace Agent.Plugins.TestResultParser.Plugin
+{
+    public class TraceLogger : ITraceLogger
+    {
+        public TraceLogger(IAgentLogPluginContext context)
+        {
+            _context = context;
+        }
+
+        #region interface implementation
+
+        /// <inheritdoc />
+        void ITraceLogger.Warning(string text)
+        {
+            _context.Output($"##vso[task.logissue type=warning;]{text}");
```

Comment:
the agent won't process any `##vso` from log plugin.
the plugin is best effort and should never add warning/error to customer's build.
the plugin allow to write to build log and agent diag log as normal output.

Reply:
yes. aware of that. But want to write in usual command form and whether agent can ignore to interpret it

Reply:
i would not write out in this format, it makes me feel there is some bug in the agent. Just write `WARNING: xxxx` and `ERROR: xxx` if you really wants warning/error in user face.